docs/sql-ref-syntax-aux-show-create-table.md

```diff
@@ -19,4 +19,39 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`SHOW CREATE TABLE` returns the `CREATE TABLE` statement that creates a given table. `SHOW CREATE TABLE` on a non-existent table throws Exception.
```

Comment:
@huaxingao Does this work on a view ? If so, can we spell it out as such ?

Reply:
Yes, this works on a view. I will add it.